apps/docs/content/guides/self-hosting/docker.mdx

```diff
@@ -127,21 +151,38 @@ The default tenant ID is `your-tenant-id`, and the default password is `your-sup
 
 By default, the database is not accessible from outside the local machine but the pooler is. You can [change this](#exposing-your-postgres-database) by updating the `docker-compose.yml` file.
 
+## Updating your services
+For security reasons, we "pin" the versions of each service in the docker-compose file (these versions are updated ~monthly). If you want to update any services immediately, you can do so by updating the version number in the docker compose file and then running `docker compose pull`. You can find all the latest docker images in the [Supabase Docker Hub](https://hub.docker.com/u/supabase).
+
+You should update your services frequently to get the latest features and bug fixes and security patches. Note that you will need to restart the services to pick up the changes, which will result in some downtime for your services.
+
+**Example**
+You'll want to update the Studio(Dashboard) frequently to get the latest features and bug fixes. To update the Dashboard:
+1. Visit the [supabase/studio](https://hub.docker.com/r/supabase/studio/tags) image in the [Supabase Docker Hub](https://hub.docker.com/u/supabase)
+2. Find the latest version (tag) number. It will look something like `20241029-46e1e40`
+3. Update the `image` field in the `docker-compose.yml` file to the new version. It should look like this: `image: supabase/studio:20241028-a265374`
```

Comment:
```suggestion
3. Update the `image` field in the `docker-compose.yml` file to the new version. It should look like this: `image: supabase/studio:20241029-46e1e40`
```

Should this match the example latest tag shown above?

Reply:
I used a different one on purpose here — showing a later date in the tag name. Thinking it would indicate "this date is later, so this must be more up to date"

maybe doesn't convey that though?

Reply:
It's sort of ambiguous, I think, I read it as "it should (now) look like this", rather than "(the old version) should look like this", but open to interpretation.

Maybe change to "it should now" with the new tag? Not a big deal though, happy to let it slide